02_TrackWeb/src/stats.ts

```
@@ -0,0 +1,221 @@
+        domainRow.className = 'expandable';
+        domainRow.innerHTML = `
+            <td>${domain}</td>
+            <td>${formatTime(data.active)}</td>
+            <td>${formatTime(data.total)}</td>
+        `;
+        tableBody.appendChild(domainRow);
+
+        // Add click handler for expansion
+        domainRow.addEventListener('click', () => {
+            const nextRow = domainRow.nextElementSibling;
+            if (nextRow && nextRow.classList.contains('url-details')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)